examples/client/Locomotion/src/pages/ActiveRide/newMap.js

```diff
@@ -221,18 +230,14 @@ export default React.forwardRef(({
         key="map"
         followsUserLocation={isUserLocationFocused}
         moveOnMarkerPress={false}
-        onRegionChangeComplete={async (event) => {
+        onPanDrag={({ nativeEvent: { coordinate: { latitude, longitude } } }) => {
+          if (!isUserLocationFocused === false) {
```

Comment:
Why not isUserLocationFocused?

Reply:
thats was already there

Reply:
it doesn't mean you don't need to change it :)

Reply:
i just didnt know if it was being used for something?